Shared/Views/Leaderboard.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Leaderboard.swift
+//  St Jude
+//
+//  Created by Ben Cardy on 29/08/2023.
+//
+
+import SwiftUI
+
+struct Leaderboard: View {
+    
+    @Environment(\.presentationMode) var presentationMode
+    
+    @Binding var campaigns: [Campaign]
```

Comment:
Is the Binding necessary here? As far as I can tell this could be just a constant